ag: add example (#2491)
tldr-pages/tldr: pages/common/ag.md

- Find files with a name matching "foo":

`ag -g {{foo}}`

> - Find files containing "foo" on directory bar/baz:

Reviewer: Perhaps this would read better?
```suggestion
- Find files containing "foo" in a specific directory:
```

> `ag foo {{bar/baz}}`

Reviewer: Add token syntax around `foo`.

Also, is `baz` a subdirectory in `bar`? I feel like using the standard path syntax may be more readable and consistent.
```suggestion
`ag {{foo}} {{path/to/directory}}`
```

> `ag {{foo}} {{path/to/folder}}`

Reviewer 2: Please make this the 2nd example.

Author: Hi @agnivade, my changes were already merged into master. Should I open a new PR for this?

Reviewer 2: Yes. Thank you.
